.github/workflows/append_release.yml

```diff
@@ -0,0 +1,189 @@
+name: 'Append Release'
+
+on:
+  workflow_call:
+    inputs:
+      release-name:
+        required: true
+        type: string
+      is-preview:
+        required: true
+        type: boolean
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download Assembled Folder
+        uses: actions/download-artifact@v2
+        with:
+          name: 'Assembled'
+          path: Package/react-native
+      - name: Compress Package react-native
+        uses: a7ul/tar-action@v1.1.3
+        id: compressreact-native
+        with:
+          command: c
+          cwd: Package/react-native/
+          files: | # all files except package.json and readme.md
+            BabylonModule.d.ts
```

Comment:
All this is to avoid copying (and overwriting) package.json. I agree it's error-prone. atm, I only see more convoluted ways (a prepass that copies files, a js script that lists files,...)

Reply:
If there were a way to exclude files for tar, would that work?

Reply:
yes, we do a PR to https://github.com/a7ul/tar-action

Reply:
Would it be okay to call tar from the shell?